controllers/controllers_test.go

```diff
@@ -18,7 +20,16 @@ import (
 const DEFAULT_ORG_ID string = "4384938490324"
 const DEFAULT_ACCOUNT_NUMBER string = "540155"
 
-func testRequest(method string, path string, accnum string, orgid string, fakeCaller func(string, string) SubscriptionsResponse, fakeBundleInfo func() []Bundle) (*httptest.ResponseRecorder, map[string]EntitlementsSection, string) {
+// func TestMain(m *testing.M) {
+// 	bundleInfo = fakeBundleInfo()
+// }
```

Comment:
Please remove the commented-out code, unless there's an important reason for it being here.

Reply:
@kruai Sorry, that was a WIP! :smile: